src/main/java/core/basesyntax/service/impl/QuantityCalculatorServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.QuantityCalculatorService;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationHandlerStrategy;
+import java.util.List;
+
+public class QuantityCalculatorServiceImpl implements QuantityCalculatorService {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private OperationHandlerStrategy operationServiceStrategy;
+    private OperationHandler operationService;
+
+    public QuantityCalculatorServiceImpl(OperationHandlerStrategy operationServiceStrategy) {
+        this.operationServiceStrategy = operationServiceStrategy;
+    }
+
+    public List<String> calculate(List<FruitTransaction> fruitTransactions) {

```

Comment:
I would suggest refactor this method totaly. The idea of using strategy pattern is get the needed service from the `strategy map` without using if else. Repeat the topic of Strategy pattern please. 
The implementation shoul looks like:

```
public void calculate(List<FruitTransaction> fruitTransactions) {
    for (FruitTransaction fruitTransaction : fruitTransactions) {
            OperationHandler handler = operationServiceStrategy.getOperationService(fruitTransaction.getOperation());
            handler.doCalculation(fruitTransaction);
        }
```
Please pay attention the method should return nothing and method `doCalculation()` should store the result of calculations to the Storage.